content/docs/command-reference/remote/modify.md

```diff
@@ -683,6 +687,78 @@ more information.
 
 <details>
 
+### Click for WebHDFS
+
+💡 WebHDFS serves as an alternative for using the same remote storage supported
+by HDFS. Read more about by expanding the WebHDFS section in
+[`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types).
```

Comment:
Again, this would prob. belong in `remote add` (maybe as the intro instead of the "regular vs web HDFS" text).